src/schema/v2/homeView/index.ts

```diff
@@ -22,11 +22,17 @@ const SectionsConnectionType = connectionWithCursorInfo({
 const SectionConnection: GraphQLFieldConfig<any, ResolverContext> = {
   type: new GraphQLNonNull(SectionsConnectionType),
   description: "A paginated list of home view sections",
-  args: pageable({}),
+  args: pageable({
+    zoneID: {
+      type: GraphQLString,
+      defaultValue: "legacy",
```

Comment:
**question**: Is this the place to encode this default value? It makes me a bit uneasy that it's rendered within the generated schema file. I wonder if that makes it more difficult to swap around without affecting clients.